app/src/actioncommands.cpp

```diff
@@ -109,6 +109,7 @@ Status ActionCommands::importMovieVideo()
     }
 
     mEditor->layers()->notifyAnimationLengthChanged();
+    mEditor->framesModified();
```

Comment:
Here, and when you call framesModified() in editor.cpp, my compiler gives a warning: Missing emit keyword. 
I'm not certain how important it is, but I generally like to avoid warnings.

Reply:
I don't get any warnings, which is odd.. however the emit keyword is simply syntactic sugar, so it shouldn't matter. I will however add it anyway.